app/models/article.rb

```diff
@@ -0,0 +1,7 @@
+class Article < ActiveRecord::Base
+	belongs_to :category
+	validates_presence_of :title, :content
+
+	scope :active, where('active = ?', true)
+  	scope :alphabetical, order('title')
```

Comment:
I really like the idea of adding articles, but I think it needs more than just the models before I can make a decision. After all, Rails is all about MVC working together, not just the M portion in isolation. If the views and controller are in working order, then I'll pull this into my project.